docs/user/site.rst

```diff
@@ -119,6 +119,8 @@ regdom \: optional
 wifi24 \: optional
     WLAN configuration for 2.4 GHz devices.
     ``channel`` must be set to a valid wireless channel for your radio.
+    ``beacon_interval`` can be specified to set a custom beacon interval in
+    milliseconds. If this is not set, the default value of 100 ms is used.
```

Comment:
Pretty sure this is wrong. It's not milliseconds, but time units. Let me check hostapd.sh.

Reply:
I just had a look at what LuCI does, and it has a very simple solution: The field is just called "Beacon Interval", without any explanation of units...